### PR TITLE
Laravel mix, bower, fix views

### PR DESCRIPTION
**Pull request by FT-Mystogan**
ready

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -23,5 +23,11 @@
...
-    "Edit Student": "Edit Student"
-}
+    "Edit Student": "Edit Student",
+    "Register Class": "Register Class",
+    "Add New Class": "Add New Class",
+    "Student Name": "Student Name",
+    "Class Name": "Class Name",
+    "Select Student": "Select Student",
+    "Select Class": "Select Class"
+}
```
Comment (ghost) [inline lang/en.json]: eof

File: lang/vi.json [hunk 2]
```diff
@@ -23,5 +24,11 @@
...
-    "Edit Student": "Sửa thông tin học sinh"
-}
+    "Edit Student": "Sửa thông tin học sinh",
+    "Register Class": "Đăng ký lớp",
+    "Add New Class": "Thêm mới lớp",
+    "Student Name": "Tên học sinh",
+    "Class Name": "Tên lớp",
+    "Select Student": "Chọn học sinh",
+    "Select Class": "Chọn lớp"
+}
```
Comment (ghost) [inline lang/vi.json]: bổ sung dòng trắng eof

**Comment by FT-Mystogan** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**